modules/ROOT/pages/microprofile-telemetry.adoc

```diff
@@ -12,19 +12,52 @@
 
+== Using OpenTelemetry ==
+
+To enable OpenTelemetry you must add one the following features `<feature>mpTelemetry-1.0</feature>` or `<feature>mpTelemetry-1.1</feature>` to your server.xml file. Then you configure one or more of: 
+
+* <<#auto, automatic instrumentation>> - which automatically traces your JAX-RS or RESTful web services.
+* <<#manual, manual instrumentation>> - which allows you to start and end telemetry spans manually.
+* <<#agent, agent instrumentation>> - which automatically adds telemetry to popular open source libraries. Agent instrumentation imposes additional limitations which are documented in <<#agent, agent instrumentation>>.
+
+OpenTelemetry requires a telemetry back end such as https://www.jaegertracing.io/[Jaeger] or https://zipkin.io/[Zipkin]. You will have to configure OpenTelemetry to use one. To do so add one of the following entries to your bootstrap.properties file:
+
+Configuring Open Telemetry to use a Jager server:
+
+[source,properties]
+----
+otel.traces.exporter=otlp
+otel.exporter.otlp.endpoint=http://localhost:4317/
+---
+
+Configuring Open Telemetry to use a Zipkin server:
+[source,properties]
+----
+otel.traces.exporter=zipkin
+otel.exporter.zipkin.endpoint=http://localhost:9411/api/v2/spans
+----
+
+=== Telemetry Versions and Java Enterprise Editions ===
+
+- `mpTelemetry-1.0` is compatible with `jakartaee-10.0` and `microProfile-6.0`
+- `mpTelemetry-1.0` is compatible with
+-- `javaee-7.0` and `microProfile-1.4`
+-- `jakartaee-8.0` and `microProfile-4.1`
+-- `jakartaee-9.1` and `microProfile-5.0`
+
 [#auto]
 == Automatic instrumentation
-With automatic instrumentation, you can observe traces without modifying the source code in your Jakarta RESTful web service (formerly JAX-RS) applications. To start emitting traces with automatic instrumentation, enable the feature:mpTelemetry[display=MicroProfile Telemetry] feature in your `server.xml` file. By default, MicroProfile Telemetry tracing is off. To enable tracing, specify the `otel.sdk.disabled=false` MicroProfile Config property and any exporter configuration that your tracing service service requires.
+With automatic instrumentation, you can observe traces without modifying the source code in your JAX-RS or Jakarta RESTful web service applications. To start emitting traces with automatic instrumentation, enable the feature:mpTelemetry[display=MicroProfile Telemetry] feature in your `server.xml` file by adding `<feature>mpTelemetry-1.0</feature>` or `<feature>mpTelemetry-1.1</feature>` to your server.xml file. By default, MicroProfile Telemetry automatic instrumentation is off. To enable tracing, specify the `otel.sdk.disabled=false` MicroProfile Config property and any exporter configuration that your tracing service service requires.
```

Comment:
Looking at the FATs, you can use manual instrumentation without otel.sdk.disabled=false so replace the word `tracing` above. 

TODO: Double check this.

Reply:
I don't _think_ that's true, which FAT test were you looking at?

Reply:
I went looking for counter examples in the publish/server. The string "sdk" does not appear inside Telemetry10Bean

Reply:
`Telemetry10Bean` is used by TelemetryBeanTest which sets this property in the config file within the application: https://github.com/OpenLiberty/open-liberty/blob/062e708ba9dbf2db640fc4c5f3f2dc4ec54dd5f9/dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryBeanTest.java#L57